apps/server/src/console/console.module.ts

```diff
@@ -1,24 +1,31 @@
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
 import { Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
 import { ConsoleWriterModule } from '@shared/infra/console/console-writer/console-writer.module';
+// TODO Have generic/abstract IDM Module Console
```

Comment:
More a note to myself. I was thinking to abstract out all administrative access (console, controller, ... ) so that we have no Keycloak exported to the outside whatsoever. Yet, on second thought, I see no benefits of having boilerplate interfaces everywhere - just more code to maintain. I'll remove that TODO.